migrate.py

```diff
@@ -677,5 +1026,10 @@ def main():
     print('======= Assumed stayed in core =======\n')
     print(yaml.dump(core))
 
+    global manual_check
+    print('======= Could not rewrite the following, please check manually =======\n')
+    for key, value in manual_check:
+        print(key + ": " + value)
```

Comment:
Maybe 
```suggestion
        logger.error(key + ": " + value)
```
to highlight it with red?